app/src/main/java/de/westnordost/streetcomplete/quests/surface/AddPitchSurface.kt

```diff
@@ -0,0 +1,49 @@
+package de.westnordost.streetcomplete.quests.surface
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.meta.updateWithCheckDate
+import de.westnordost.streetcomplete.data.osm.osmquest.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+
+
+class AddPitchSurface : OsmFilterQuestType<SurfaceAnswer>() {
+        override val elementFilter = """
+        ways with leisure=pitch
+        and sport ~ soccer|tennis|basketball|equestrian|athletics|volleyball|skateboard|bmx|american_football|rugby_union
```

Comment:
Why limit the sports?

Reply:
I got scared by by https://taginfo.openstreetmap.org/tags/sport=table_tennis (confusing), https://taginfo.openstreetmap.org/tags/sport=shooting#combinations (where quest would be pointless), https://taginfo.openstreetmap.org/tags/sport=beachvolleyball#combinations (here it would be pointless, it should be always sand), and baseball for example has clay + grass pitches (multiselect of surfaces is in principle possible).

https://taginfo.openstreetmap.org/tags/sport=golf#combinations should be both pointless and requiring multiple surfaces, https://taginfo.openstreetmap.org/tags/sport=boules is always surface=compacted etc

Reply:
Right, but then your list of sports is a little small. Maybe have a negative list instead?

Reply:
Which sports are missing?

For negation it would be necessary to add syntax that would have regexp matching without adding `^` `$` (to skip also say `shooting;archery`, `ice_skating;ice_hockey`, `cricket;australian_football`, `skating;hockey`, `discus_throw;hammer_throw` etc).

Also, based on my experience with blacklisting values - that is just never ending quest for adding new variables, so I would strongly prefer whitelisting. I used blacklist in some places in my code and it was very irritating in its outcome.

Reply:
> Which sports are missing?

Uf, I will not give you a list, but just skimming that I noticed canadian football, australian football, softball, baseball, cricket, that other rugby, gaelic games and sepak takraw.

Reply:
> For negation it would be necessary to add syntax that would have regexp matching without adding ^ $

`.*ice_hockey|ice_skating|table_tennis.*`

Reply:
Baseball was skipped deliberately as it is standard that field will have parts of different surfaces (grass + clay), at least a bit better ones (according to what people from USA told me).

As I understand the same is true for cricket: https://en.wikipedia.org/wiki/Cricket#/media/File:Muralitharan_bowling_to_Adam_Gilchrist.jpg (though looking again at actual tagging, it seems to be done differently? Added after all)

Others are added already or I added them right now.

And blacklist would need to be a bit longer what in turn convinces me that whitelist was not so bad idea...

`.*multi|ice_hockey|([^_]^)hockey|ice_skating|table_tennis|baseball|beachvolleyball|climbing|shooting|archery|scuba_diving|canoe|skiing|free_flying.*`

(for example there are some marked canoe pitches in places where boats are stored or member meet, asking about ice_skating or beachvolleyball is quite pointless etc)

Reply:
And further ones that would end on a blacklist: 10pin model_aerodrome skating billiards trampoline 9pin rowing sailing climbing_adventure long_jump ice_stock ski_jumping (I have not checked whatever this is tagged on `leisure=pitch` but after canoe pitches on land I assume that at least some of them are).

Not on pitches for now but maybe would be better to blacklist just in case: orienteering

Not sure what to do with bullfighting and chess (chess on areas is a tagging for oversized chess boards where asking about surface would make sense, but question there wold be confusing)

On additional check: everyone tags baseball pitches with just a single value, so it is back on a whitelist and should be dropped from blacklist.

Reply:
In the end I think that having exemption list and valid list is equally irritating, and false negatives are better than false positives.